Server/bkr/server/config/app.cfg

```diff
@@ -100,7 +100,7 @@ beaker.inventory_osmajors = ['RedHatEnterpriseLinux8', 'RedHatEnterpriseLinux7',
 # gzip_filter.on = True
 # gzip_filter.mime_types = ["application/x-javascript", "text/javascript", "text/html", "text/css", "text/plain"]
 
-beaker.bz_create_link = "https://bugzilla.redhat.com/enter_bug.cgi?product=Beaker"
+beaker.bz_create_link = "https://github.com/beaker-project/beaker/issues/"
```

Comment:
Why not beaker.[gh|issue]_create_link ?
(and then update master.kid to match it)